templates/rke2-replace-node/overlay/opt/corral/rke2/delete_nodes.sh

```diff
@@ -0,0 +1,6 @@
+#1/bin/bash
```

Comment:
```suggestion
#!/bin/bash
```
Also should be `delete-nodes.sh`